clickhouse_sqlalchemy/drivers/http/escaper.py

```diff
@@ -38,9 +38,11 @@ def escape_number(self, item):
         return item
 
     def escape_date(self, item):
+        # XXX: shouldn't this be `toDate(...)`?
```

Comment:
Escaping is used in query bind parameters. `SELECT %(param)s` renders into `SELECT '2020-01-27'` in query that is sent to ClickHouse server.

Reply:
In those terms, the query could be either `SELECT %(param)s`, or `SELECT toDate(%(param)s)`.

CH does a lot of implicit casting, but in general, the latter is likely more correct (sample case would be "select dt1 + dt2").

However, there might be some opposite cases, so I haven't touched it yet.

Reply:
Yes, you're right. This should render into `select dt1 + dt2`. PG example:

```python
In [1]: import psycopg2 
   ...: conn = psycopg2.connect(...)                                                                                                           

In [2]: cur = conn.cursor()                                                                                                                                                     

In [3]: from datetime import date                                                                                                                                               

In [4]: cur.execute('SELECT %(a)s + %(b)s', {'a': date.today(), 'b': date.today()})                                                                                             
---------------------------------------------------------------------------
UndefinedFunction                         Traceback (most recent call last)
<ipython-input-5-375ff6fd4f34> in <module>
----> 1 cur.execute('SELECT %(a)s + %(b)s', {'a': date.today(), 'b': date.today()})

UndefinedFunction: operator does not exist: date + date
LINE 1: SELECT '2020-02-10'::date + '2020-02-10'::date
```

I think it's good idea to make as much similar behavior with another dialects as possible. PostgreSQL is the closest dialect to this one.